.changeset/shaggy-falcons-sparkle.md

```diff
@@ -0,0 +1,12 @@
+---
+"@pnpm/audit": patch
+"@pnpm/core": patch
+"@pnpm/fetch": patch
+"@pnpm/npm-resolver": patch
+"@pnpm/package-requester": patch
+"@pnpm/plugin-commands-audit": patch
+"@pnpm/plugin-commands-env": patch
+"@pnpm/tarball-fetcher": patch
+---
+
+Upgrade nock to v13.
```

Comment:
you don't need a changeset for these changes because it only affects tests. We won't release new versions due to changes to tests or dev dependencies.

Reply:
Thanks for letting me know, I assumed so.

It seems the CI pipelines will all passing, so I will push another commit with removal of changeset, then should be good to merge.